dotCMS/src/main/java/com/dotcms/graphql/datafetcher/ContentMapDataFetcher.java

```diff
@@ -0,0 +1,20 @@
+package com.dotcms.graphql.datafetcher;
+
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.util.Logger;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+
+public class ContentMapDataFetcher implements DataFetcher<Object> {
+    @Override
+    public Object get(final DataFetchingEnvironment environment) throws Exception {
+        try {
+            final Contentlet contentlet = environment.getSource();
+            final String key = environment.getArgument("key");
+            return contentlet.get(key);
+        } catch (Exception e) {
```

Comment:
Codacy found an issue: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6090726)